.changeset/khaki-bananas-know.md

```diff
@@ -0,0 +1,13 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+Dependencies `ember-a11y-refocus` and `@glimmer/component` upgraded.
```

Comment:
[suggestion] I would probably split this in two entries, and add which version (at least the `major.minor`)

Reply:
makes sense! I made the upgrades major and the type fixes as a patch (I checked and it doesnt seem like anyone is using those args)